ext/date/php_date.stub.php

```diff
@@ -73,7 +73,7 @@ function date_isodate_set(DateTime $object, int $year, int $week, int $day = 1):
 
 function date_timestamp_set(DateTime $object, int $timestamp): DateTime {}
 
-function date_timestamp_get(DateTimeInterface $object): int|false {}
+function date_timestamp_get(DateTimeInterface $object): int {}
```

Comment:
This is not correct, it can still return false.

Reply:
This hasn't been fixed yet, it can still return false.

Reply:
I don't see how it can after you told me to convert the false return of the epoch not fitting in a zend long to ValueError, or am I missing something in regards to the OO API?